app/uk/gov/hmrc/onestopshopforexrates/config/ModuleBindings.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.onestopshopforexrates.config
+
+import play.api.{Configuration, Environment}
+import play.api.inject.{Binding, Module}
+import uk.gov.hmrc.onestopshopforexrates.scheduler.jobs.{RetrieveAndSendForexDataJob, RetrieveAndSendForexDataJobImpl}
+import uk.gov.hmrc.onestopshopforexrates.services.{ExchangeRatesService, ExchangeRatesServiceImpl}
+
+import java.time.{Clock, ZoneOffset}
+
+class ModuleBindings extends Module {
+
+  override def bindings(environment: Environment, configuration: Configuration): Seq[Binding[_]] = Seq(
+    bind(classOf[Clock]).toInstance(Clock.systemDefaultZone.withZone(ZoneOffset.UTC)),
+    bind[ExchangeRatesService].to[ExchangeRatesServiceImpl].eagerly(),
```

Comment:
Why do we need to eagerly inject this?

Reply:
We actually didn't need eager binding of the service, just the job itself, thanks for spotting this! Fixed now